Sources/RealDeviceMapLib/Modell/Account.swift

```diff
@@ -609,7 +611,8 @@ public class Account: WebHookEvent {
                 \(spinSQL)
                 \(cooldownSQL)
                 \(lockoutSQL)
-            ORDER BY level DESC, last_used_timestamp DESC
+            ORDER BY level \(Account.lastRecentlyUsed ? "DESC" : "ASC"),
```

Comment:
I think it's okay to remove the ORDER BY level in total... we want to use last recently used or not... ordering by level would prefer always highest or lowest level first

Reply:
I agree. I don't see a problem in removing it.
Have pushed a change.